packages/nextjs/package.json

```diff
@@ -14,20 +14,24 @@
     "vercel:yolo": "vercel --build-env NEXT_PUBLIC_IGNORE_BUILD_ERROR=true"
   },
   "dependencies": {
+    "@apollo/client": "^3.9.4",
     "@ethersproject/providers": "^5.7.2",
     "@heroicons/react": "^2.0.11",
     "@rainbow-me/rainbowkit": "1.3.0",
     "@uniswap/sdk-core": "^4.0.1",
     "@uniswap/v2-sdk": "^3.0.1",
+    "@urql/core": "^4.2.3",
```

Comment:
It seems like URQL is not used, so why install it?

Reply:
Thanks! I missed that - cleaned this up here https://github.com/scaffold-eth/scaffold-eth-2/pull/722